src/main/java/seedu/toluist/controller/SortController.java

```diff
@@ -21,10 +22,15 @@
 public class SortController extends Controller {
     private static final String RESULT_MESSAGE = "List is now sorted by: %s.";
     private static final String ERROR_MESSAGE = "Unable to sort by: %s";
+    private static final String NO_VALID_KEYWORD_MESSAGE = "No valid keyword entered."
```

Comment:
`MESSAGE` should be in front to follow the coding standard. Make sure you have everything changed